pkg/commands/types.go

```diff
@@ -246,44 +243,3 @@ func (params *KnParams) RestConfig() (*rest.Config, error) {
 
 	return config, nil
 }
-
-// GetClientConfig gets ClientConfig from KubeCfgPath
-func (params *KnParams) GetClientConfig() (clientcmd.ClientConfig, error) {
```

Comment:
Thanks, for cleaning this part. It seems like a very old piece, that doesn't make a lot of sense here.

Reply:
Yep. Now, the kubernetes related parsms are in their own struct, which makes far more sense.

BTW. I love the struct composition in Go. Such a great idea, they had...

Reply:
Ops, in my first comment I thought it's located in different file. I.e. that an isolated command tried to override the config loading. Anyway, it makes sense here, but it's fine to have it extracted as you did. All good here! :)